package.json

```diff
@@ -36,7 +36,29 @@
   "aiKey": "INSERTAIKEY",
   "main": "./out/src/extension.js",
   "activationEvents": [
-    "*"
+    "onView:truffle-vscode.truffle",
+    "onView:truffle-vscode.explorer-view",
+    "onCommand:truffle-vscode.showWelcomePage",
+    "onCommand:truffle-vscode.refresh",
+    "onCommand:truffle-contract.copyByteCode",
+    "onCommand:truffle-contract.copyDeployedByteCode",
+    "onCommand:truffle-contract.copyABI",
+    "onCommand:truffle-vscode.createLocalUI",
+    "onCommand:truffle-vscode.newSolidityProject",
+    "onCommand:truffle-vscode.buildContracts",
+    "onCommand:truffle-vscode.deployContracts",
+    "onCommand:truffle-vscode.connectProject",
+    "onCommand:truffle-vscode.copyRPCEndpointAddress",
+    "onCommand:truffle-vscode.createProject",
+    "onCommand:truffle-vscode.disconnectProject",
+    "onCommand:truffle-vscode.debugTransaction",
+    "onCommand:truffle-vscode.signInToInfuraAccount",
+    "onCommand:truffle-vscode.signOutOfInfuraAccount",
+    "onCommand:truffle-vscode.startGanacheServer",
+    "onCommand:truffle-vscode.stopGanacheServer",
+    "onCommand:truffle-vscode.views.explorer.refreshExplorer",
+    "onCommand:truffle-vscode.views.deployments.refresh",
+    "onDebug"
```

Comment:
Should we reintroduce the full list of events[1]? These are the events that are missing

```json
    "onCommand:truffle-vscode.createContract",
    "onCommand:truffle-vscode.views.dashboard.startDashboardServer",
    "onCommand:truffle-vscode.views.dashboard.stopDashboardServer",
    "onCommand:truffle-vscode.views.dashboard.restartDashboardServer",
    "onCommand:truffle-vscode.views.dashboard.openDashboard",
    "onCommand:truffle-vscode.views.dashboard.copyRPCEndpointAddress",
```

---

https://github.com/trufflesuite/vscode-ext/pull/136/files

Reply:
Hey @acuarica, Thank you :)

I'm gonna put those ones

Reply:
@acuarica done!